<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRCode enregistrer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54,54,54)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menubar">
     <property name="styleSheet">
      <string notr="true">background-color:#FFF</string>
     </property>
     <widget class="QMenu" name="menuMenu">
      <property name="title">
       <string>Menu</string>
      </property>
      <addaction name="actionVoir_les_QRcode_enregistrer"/>
      <addaction name="actionGenerate_QRcode"/>
     </widget>
     <addaction name="menuMenu"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:rgb(225, 225, 225); font-size:28pt;</string>
     </property>
     <property name="text">
      <string>Vos qrcodes :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>806</width>
        <height>502</height>
       </rect>
      </property>
      <widget class="QLabel" name="qrcode_left">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>256</width>
         <height>256</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="qrcode_right">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>130</y>
         <width>256</width>
         <height>256</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="url_left">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>256</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="url_right">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>70</y>
         <width>256</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionVoir_les_QRcode_enregistrer">
   <property name="text">
    <string>Voir les QRcode enregistrer</string>
   </property>
  </action>
  <action name="actionGenerate_QRcode">
   <property name="text">
    <string>Generate QRcode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
